src/main/java/org/skriptlang/skript/elements/expressions/displays/text/ExprTextDisplayOpacity.java

```diff
@@ -0,0 +1,103 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.elements.expressions.displays.text;
+
+import org.bukkit.entity.Display;
+import org.bukkit.entity.TextDisplay;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Text Display Opacity")
+@Description({
+	"Returns or changes the opacity of <a href='classes.html#display'>text displays</a>.",
+	"Values are between -127 and 127. The value of 127 represents it being completely opaque."
+})
+@Examples("set the opacity of the last spawned text display to -1 # Reset")
+@Since("INSERT VERSION")
+public class ExprTextDisplayOpacity extends SimplePropertyExpression<Display, Byte> {
+
+	static {
+		if (Skript.isRunningMinecraft(1, 19, 4))
+			registerDefault(ExprTextDisplayOpacity.class, Byte.class, "[display] opacity", "displays");
+	}
+
+	@Override
+	@Nullable
+	public Byte convert(Display display) {
+		if (!(display instanceof TextDisplay))
+			return null;
+		return ((TextDisplay) display).getTextOpacity();
+	}
+
+	@Nullable
+	public Class<?>[] acceptChange(ChangeMode mode) {
+		return CollectionUtils.array(Number.class);
```

Comment:
Is remove all really a valid changer here? I suppose it works, but doesn't make much sense to `remove all 1 from opacity of x`.
Also, will default to supporting any new change modes. Not sure if that's a concern.